core/sds-aspect-model-java-generator/src/main/resources/java-pojo-hashcode-method-lib.vm

```diff
@@ -0,0 +1,11 @@
+#macro( javaPojoHashCodeMethod )
+@Override
+public int hashCode() {
+    return Objects.hash(
+    #foreach( $property in $element.getProperties() )
```

Comment:
Missing super.hashCode call

Reply:
The call to `super.equals` is also missing. I will update the logic for both methods to have consistent `equals` and `hashcode` implementations.

Reply:
The classes are never generated with any explicit superclass, so what is the point?

Reply:
In case an Entity extends another Entity, then the corresponding class is generated with a superclass. Lets say we the `Car` Entity which extends the `Vehicle` Entity. Then the class fro the `Car` Entity is generated as
`public class Car extends Vehicle`.

Reply:
You're right, makes sense!